front/src/modules/comments/components/Timeline.tsx

```diff
@@ -0,0 +1,288 @@
+import React from 'react';
```

Comment:
I won't refactor this component as very likely to evolve in the upcoming days + tech debt very low as easy to refactor